src/expr/src/expr/expr_unary.rs

```diff
@@ -289,6 +289,11 @@ pub fn new_unary_expr(
             return_type,
             ascii,
         )),
+        (ProstType::CharLength, _, _) => Box::new(UnaryExpression::<Utf8Array, I64Array, _>::new(
```

Comment:
nit: 
By https://www.postgresql.org/docs/9.1/functions-string.html and https://www.postgresql.org/docs/current/datatype.html,
it seems the data type of the return value should be `signed four-byte integer`